be/src/fs/s3/poco_common.cpp

```diff
@@ -0,0 +1,100 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "fs/s3/poco_common.h"
+
+#include <Poco/Exception.h>
+#include <fmt/format.h>
+
+#include <memory>
+#include <sstream>
+
+namespace starrocks::poco {
+
+bool isHTTPS(const Poco::URI& uri) {
+    if (uri.getScheme() == "https")
```

Comment:
not sure if this is a case-sensitive or not. Suppose it should be case-insensitive. better double check it.

Reply:
will check it

Reply:
I have checked this, it ensured by aws-sdk.